akka_microservices/ibex35Microservice/src/main/scala/com/tapiporla/microservices/retrievers/indices/stock/StockHistoricManager.scala

```diff
@@ -79,20 +79,20 @@ class Ibex35HistoricManager extends Retriever with Stash {
   def ready(lastUpdated: Option[DateTime]): Receive = {
 
     case UpdateHistoricData =>
-      log.info(s"Time to update historic data from Ibex35")
+      log.info(s"Time to update historic data from ${stockName}")
       lastUpdated map { date =>
-        scrapyDAO ! RetrieveIbexDataFrom(date)
+        scrapyDAO ! RetrieveStockDataFrom(date)
       } getOrElse
-        scrapyDAO ! RetrieveAllIbexData
+        scrapyDAO ! RetrieveAllStockData
 
-    case IbexDataRetrieved(historicData) =>
-      log.info(s"Data retrieved from Ibex35 to be updated ${historicData.length}")
+    case StockDataRetrieved(historicData) =>
+      log.info(s"Data retrieved from ${stockName} to be updated ${historicData.length}")
```

Comment:
You can omit braces when mixing simple expressions and string interpolation

Reply:
Copy paste is not my best friend xD

Done